pkg/scheduler/plugins/resourcequota/resourcequota.go

```diff
@@ -0,0 +1,79 @@
+package resourcequota
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+	quotav1 "k8s.io/kubernetes/pkg/quota/v1"
+
+	scheduling "volcano.sh/apis/pkg/apis/scheduling/v1beta1"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/plugins/util"
+)
+
+// PluginName indicates name of volcano scheduler plugin.
+const PluginName = "resourcequota"
+
+// resourceQuota scope not supported
+type resourceQuotaPlugin struct {
+	// Arguments given for the plugin
+	pluginArguments framework.Arguments
+}
+
+// New return resourcequota plugin
+func New(arguments framework.Arguments) framework.Plugin {
+	return &resourceQuotaPlugin{
+		pluginArguments: arguments,
+	}
+}
+
+func (rq *resourceQuotaPlugin) Name() string {
+	return PluginName
+}
+
+func (rq *resourceQuotaPlugin) OnSessionOpen(ssn *framework.Session) {
+	pendingResources := make(map[string]v1.ResourceList)
+
+	ssn.AddJobEnqueueableFn(rq.Name(), func(obj interface{}) int {
+		job := obj.(*api.JobInfo)
+
+		resourcesRequests := job.PodGroup.Spec.MinResources
```

Comment:
we should check if `resourcesRequests == nil` in the logic behind

Reply:
done